docs(readme): clarify live vs community networks (#126)
interledgerjs/moneyd: README.md

- [Sending Payments](#sending-payments)
- [Connector List](#connector-list)

## Interledger Networks

> When moneyd was created, the Interledger livenet was a community-run network for research and development

Reviewer: ```suggestion
When moneyd was created, what we referred to as the _Interledger livenet_ was a community-run network for research and development
```

> purposes. Now there is a well-established Interledger livenet based around wallet providers such as

Reviewer: ```suggestion
purposes. Now there is a well-established Interledger network based around wallet providers such as
```

> [Gatehub](https://gatehub.net), [XRPTipBot](https://xrptipbot.com), and [Stronghold](https://stronghold.co/).

Reviewer: ```suggestion
[Gatehub](https://gatehub.net), [XRPTipBot](https://xrptipbot.com), and [Stronghold](https://stronghold.co/), which we call the today's Interledger livenet.
```

Today's Interledger livenet makes it easy for anyone to get a payment pointer to receive Interledger payments, and
has handled billions of micropayments. However, the wallets which now make up the livenet have not yet exposed

> moneyd-like direct Interleger access.

Reviewer 2: ```suggestion
moneyd-like direct Interledger access.
```

> As a result, moneyd will only run against a community network of connectors, not the Interledger livenet.

Reviewer 2: ```suggestion
As a result, moneyd will only connect to a community network of connectors, not the Interledger livenet.
```

or "peer with".

> It's still a useful tool for experimenting with Interledger, but it is not currently suitable for any production

Reviewer 2: ```suggestion
Moneyd is still a useful tool for experimenting with Interledger, but it is not currently suitable for any production
```

Clarify, because `It` could refer to either "moneyd" or "the Interledger livenet" here.

> use cases.

Reviewer 2: Consider noting that the community network does use real money. Without clarification, "not currently suitable for any production use cases" implies the opposite.
